uber/templates/regform.html

```diff
@@ -415,10 +422,19 @@
 <div class="form-group">
     <label for="email" class="col-sm-3 control-label">Email Address</label>
     <div class="col-sm-6">
-        <input type="email" type="email" name="email" id="email" value="{{ attendee.email }}" class="form-control" placeholder="Email Address">
+        <input type="email" name="email" id="email" value="{{ attendee.email }}" class="form-control" placeholder="Email Address">
     </div>
 </div>
 
+{% if is_prereg_confirm_email_enabled %}
+  <div class="form-group">
+      <label for="confirm_email" class="col-sm-3 control-label">Confirm Email</label>
+      <div class="col-sm-6">
+          <input type="email" name="confirm_email" id="confirm_email" value="{{ attendee.email }}" class="form-control" placeholder="Type Email Address Again">
```

Comment:
Placeholder wording suggestion: "Confirm your email address" is pretty standard for these double-type forms.